docs/modeling/processes/bpmn/user.md

```diff
@@ -3,26 +3,133 @@ Title: User tasks
-* Candidate user: a list of users that may claim the task. 
-* Candidate group: a group(s) of users that can claim the task. 
+```xml
+<bpmn2:userTask id="UserTask_0gpdh83" name="Order" activiti:formKey="form-38098a3e-bff1-46cb-ba0f-0c94fdb287ed" activiti:assignee="${userDetails.username}" activiti:dueDate="2020-01-01T01:00:00" activiti:priority="2">
+	<bpmn2:documentation>A form to choose the flavor of ice cream.</bpmn2:documentation>
+	<bpmn2:incoming>SequenceFlow_02eaofe</bpmn2:incoming>
+	<bpmn2:outgoing>SequenceFlow_14ma5mo</bpmn2:outgoing>
+</bpmn2:userTask>
+``` 
+
+## Properties 
+
+### Basic properties
+The basic properties for a user task are: 
+
+| Property | Description | Example | Required | 
+| -------- | ----------- | ------- | -------- | 
+| `id` | The unique identifier for the user task. This is system generated and cannot be altered | UserTask_0gpdh83 | Yes |
+| `name` | The name of the user task. This will display on the user task in the process diagram | Flavor order | No |
+| `documentation` | A free text description of what the user task does | A form to choose the flavor of ice cream.  | No |
+
+The ID and name of a user task are set as XML attributes of the `userTask`. Documentation is a sub-element of `userTask`, for example: 
+
+```xml
+<bpmn2:userTask id="UserTask_0gpdh83" name="Order">
+	<bpmn2:documentation>A form to choose the flavor of ice cream.</bpmn2:documentation>
+</bpmn2:userTask>
+```
+
+### Assignment
+The users or groups that are able to complete a task. A single user can be assigned or candidates can be set. Candidates are a list of users or groups that may [claim a task](../../../workspace/tasks.md#claiming-a-task) at runtime. A single user or candidates must be set for a user task.  
+
+A single assignee is set in the XML attribute `activiti:assignee` of the `userTask`, for example: 
+
+```xml
+<bpmn2:userTask activiti:assignee="hruser"> 
+```
+
+Candidates are set in the XML attribute `activiti:candidateGroups` of the `userTask`, for example:
+
+```xml
+<bpmn2:userTask activiti:candidateGroups="hrgroup,${groupDetails.groupNames}">
+```
+
+Users and groups can be set from three different sources: 
+
+* **Static** values are a free text field that has no validation against users in the [Identity Service](../../../administrator/identity/service.md). The text entered will require an exact match to a `username` in the Identity Service for the user task to be correctly assigned at runtime.   
+
+* **Identity** allows for users and groups to be searched in the [Identity Service](../../../administrator/identity/service.md) and selected for the assignment.
+
+* **Expression** allows for an expression using variables to be used to select users and groups for the assignment. Expressions can be a simple process variable such as `${userToAssign}` or an expression such as `${userDetails.username}` that uses a process variable of type JSON. A JSON editor is provided for creating expressions for assignment.
```

Comment:
can you say that we show the editor only if there are process variable available?

Reply:
Updated to note that a variable is required for it to be displayed